busted/runner.lua

```diff
@@ -41,6 +41,11 @@ return function(options)
     exit(1, forceExit)
   end
 
+  if cliArgs.help then
+    io.stdout:write(cliArgs.helpText)
```

Comment:
just checking: does the help text include a trailing newline? `io.stdout:write()` will not automatically append one like `print()` does.

Reply:
Good catch, it does *not* include a trailing newline and is used elsewhere inline so it doesn't make sense to add it to the content. I just addressed this by explicitly concating a newline to it as we write it out, and also normalized away the one use of `print()` instead of `io.<streat>:write()` like we are using everywhere else in this module.